assets/javascripts/lib/apply-spoiler.js

```diff
@@ -50,14 +64,12 @@ function _setSpoilerVisible(element) {
   const spoilerVisibleAttributes = {
     "data-spoiler-state": "revealed",
     "aria-expanded": true,
-    "aria-label": I18n.t("spoiler.label.hide"),
```

Comment:
Since this is no longer used here and anywhere else in the plugin, I think we can also safely remove it from the locale files.

``` diff
      label:
        show: "Show hidden content"
-       hide: "Hide content"
```

Reply:
Just in English or all locales?

Reply:
Just in `config/locales/client.en.yml` should be fine. I believe Crowdin will automatically update the rest.